forest/shared/src/macros.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::time::Duration;
+
+pub const DEFAULT_RETRIES: i32 = 3;
+pub const DEFAULT_DELAY: Duration = Duration::from_secs(60);
+
+#[macro_export]
```

Comment:
Even though the snapshot fetch can get tricky to test, I believe this macro would be trivial to cover with unit tests. What do you think?

Reply:
yeah I can add it

Reply:
Fantastic, thanks.